### DISCUSSION TITLE
Intensity vs particular Euler angle plot along a particular line

### DISCUSSION THREAD

**Discussion by Samarendra0101** [mtex-toolbox/mtex#1733] [Ask Anything]
How can I plot an intensity (f(g)) vs particular Euler angle (say Φ1 or Φ or Φ2) along a particular line from an ODF? 
For instance I want to plot f(g) vs. Φ1 along the line, Φ = 45°, Φ2 = 0° , what is the command for that? I've also attached an image for reference.
![intensity vs phi1](https://user-images.githubusercontent.com/106327961/217613614-e54bfce9-1d76-4f3c-adc1-5496f556a63f.jpg)

**Comment by Liujianhua00** [comment]
I'm also quite interested in this question, and I find that in fact it's a line plots or a one-dimensional ODF plots, in the documentation of MTEX, it provides an example of line plot. And I give it a try, here is the code, but I'm not sure it's right or not, maybe you can check it.

`ori1 = orientation.byEuler(0*degree,0*degree,0*degree,cs);
ori2 = orientation.byEuler(90*degree,0*degree,0*degree,cs);
f = fibre(ori1,ori2);
plot(odf,f,'linewidth',2,'linestyle','-.')`

**Reply by Liujianhua00** [reply]
The ori1 and ori2 are starting orientation and ending orientation (defined by Euler angles), respectively. So you can change these three angles to get the range you want.

**Reply by Samarendra0101** [reply]
Thank you so much for your response. This command is working for me. Thank you again for the help
